mu-plugins/000-composer.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Composer autoloader for create-wordpress-project
+ *
+ * This site depends on Composer autoloading to work, so this file needs to be first in the mu-plugins load sequence.
+ *
+ * @package wpr
+ */
+
+require_once dirname( __DIR__ ) . '/vendor/autoload.php';
```

Comment:
Could be nice to have a friendly version of this from https://github.com/alleyinteractive/wp-starter-project/blob/main/client-mu-plugins/0-composer.php

Reply:
Nice, I totally forgot that existed. I'll port it over.